src/client/components/household-information/DeductibleExpensesSection.jsx

```diff
@@ -56,7 +56,6 @@ class DeductibleExpensesSection extends React.Component {
       content = (<fieldset>
         <legend>Previous calendar year deductible expenses</legend>
         {notRequiredMessage}
```

Comment:
Good point, what we're passing is kind of messed up. I'm just going to remove these notRequiredMessages for now because I think we're taking care of this in the FinancialDisclosure language.